salt/_states/metalk8s_kubernetes.py

```diff
@@ -472,6 +472,58 @@ def namespace_present(name, **kwargs):
     return ret
 
 
+def namespace_annotation_present(
+        name,
+        annotation_key,
+        annotation_value,
+        **kwargs):
+    '''
+    Ensures that the namespace is patched.
+
+    name
+        The name of the namespace
+
+    annotation_key
+        Key of the annotation
+    annotation_value
+        Value of the annotation
+    '''
+
+    ret = {'name': name,
+           'changes': {},
+           'result': False,
+           'comment': ''}
+
+    annotation = __salt__['metalk8s_kubernetes.show_namespace'](
+        name, **kwargs
+    )
+
+    if annotation_key not in annotation:
+        ret['comment'] = 'The annotation to be added does not exist'
```

Comment:
I know this line is outside of the `if __opts__['test']:` in the `node_annotation_present` but for me you should put it in the `if` section

Reply:
Also keep the original message it's better I think because here the sentence sound like it's a problem.
`The annotation is going to be set` better IMO